client/my-sites/stats/info-panel/index.jsx

```diff
@@ -61,7 +61,7 @@ class StatsInfoPanel extends React.PureComponent {
 									href="http://wordpress.com/support/stats/#referrers"
 								>
 									<Gridicon icon="info-outline" />
-									{ this.props.translate( 'About Referrers' ) }
+									{ this.props.translate( 'About referrers' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.